src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -61,12 +61,12 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
   const expectedTxOutcome = useAppSelector(VaultsSelectors.selectExpectedTxOutcome);
   const expectedTxOutcomeStatus = useAppSelector(VaultsSelectors.selectExpectedTxOutcomeStatus);
   const actionsStatus = useAppSelector(LabsSelectors.selectSelectedLabActionsStatusMap);
+  const selectedTokenMap = useAppSelector(TokensSelectors.selectTokensMap);
 
   const sellTokensOptions = selectedLab
     ? [selectedLab.token, ...userTokens.filter(({ address }) => address !== selectedLab.token.address)]
     : userTokens;
```

Comment:
nit: It appears to me that this belongs in the `getSelectedSellToken()`, we are doing the logic here and then giving it as an argument but what do you think we just move this logic into the `getSelectedSellToken()`?

Then we could potentially remove it from the checks down below, like:

```typescript
  if (!selectedLab || !selectedSellTokenAddress || !selectedSellToken || !sellTokensOptions) {
    return null;
  }
```

That for example could just become

```typescript
  if (!selectedLab || !selectedSellTokenAddress || !selectedSellToken) {
    return null;
  }
```